tests/flow/test_ts_resp3.py

```diff
@@ -24,11 +24,13 @@ def test_resp3(env):
         assert res == [1000, 5.0]
         res = r.execute_command('ts.info', t1, 'DEBUG')
         res.pop(b'memoryUsage')
+        default_duplicate_policy = env.cmd("config", "get", "ts-duplicate-policy")[b"ts-duplicate-policy"]
```

Comment:
What is it that we're comparing here? we get the same information from the database in two different ways, obviously the currently existing default duplicate policy is the currently used duplicate policy. is that it? we're not checking that it is some expected value?

Reply:
Well, no.
This was exactly the fix: make sure that a newly added time-series gets the default duplicate policy when the creation command (either TS.ADD or TS.CREATE) doesn't include an explicit `DUPLICATE_POLICY`.